### ISSUE TITLE
PTSD 2/2 accessibility audit

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#94612]
## Issue Description
Conduct an accessibility audit of the following pages:

ptsd-secondary-incident-date
ptsd-secondary-incident-unit-assignment
ptsd-secondary-incident-location
ptsd-secondary-incident-support
ptsd-secondary-incident-description
ptsd-secondary-other-sources
ptsd-secondary-other-sources-help
ptsd-secondary-permission-notice
ptsd-secondary-authorities
ptsd-secondary-upload-supporting-sources-choice
ptsd-secondary-upload-supporting-sources
ptsd-781a-additional-events
ptsd-781a-upload
ptsd-assault-additional-incident
ptsd-781a-changes-introduction
ptsd-781a-physical-changes
ptsd-781a-metal-changes
ptsd-781a-work-changes
ptsd-781a-social-changes
ptsd-781a-additional-changes

---
## Tasks
- [ ] Conduct the accessibility audit as outlined in the Epic for the pages listed above
- [ ] Record bugs & violations in Google Sheets

## Acceptance Criteria
- [ ] An accessibility audit is performed for the pages listed above
- [ ] Results are documented in Google Sheets
- [ ] A summary is shared with our stakeholders during Sprint Review
